src/models/events/push.rs

```diff
@@ -56,6 +56,14 @@ pub enum SlackEventCallbackBody {
     AppUninstalled(SlackAppUninstalledEvent),
     LinkShared(SlackLinkSharedEvent),
     EmojiChanged(SlackEmojiChangedEvent),
+    MemberJoinedChannel(MemberJoinedChannelEvent),
```

Comment:
For consistency's sake, it would be good to call those types starting with `Slack....` as we have for previous events.

Reply:
Oh yeah sorry. I'll fix that quickly

Reply:
Fixed